pillar/params.sls

```diff
@@ -101,6 +101,21 @@ proxy:
   no_proxy:       ''
   systemwide:     'true'
 
+# Kubernetes is designed to work with different Clouds such as Google Compute Engine (GCE), 
+# Amazon Web Services (AWS), and OpenStack; therefore, different load balancers need to be created 
+# on the particular Cloud for the services. This is done through a plugin for each Cloud.
+# https://github.com/kubernetes/kubernetes/blob/release-1.7/pkg/cloudprovider/README.md
+cloud:
+  provider:     ''
+  config:
+    auth_url:   ''
+    domain:     ''
+    project:    ''
+    region:     ''
+    username:   ''
+    password:   ''
+    subnet:     ''
```

Comment:
Each cloud provider has different options that are needed, these should be scoped to "openstack"